src/ResultsActivity/ResultsActivity.js

```diff
@@ -37,6 +37,12 @@ const StackedTotals = styled.div`
   flex-direction: column;
 `
 
+const HideForPrint = styled.span`
```

Comment:
Duplicated, for now, need to clean up the duplication in both the house block and activity lists, either by combining them or moving out common functions and styles out into a common file.